src/rpc/mining.cpp

```diff
@@ -434,6 +435,9 @@ static RPCHelpMan getmininginfo()
     obj.pushKV("networkhashps",    getnetworkhashps().HandleRequest(request));
     obj.pushKV("pooledtx",         (uint64_t)mempool.size());
     obj.pushKV("chain",            Params().NetworkIDString());
+    if (Params().GetConsensus().signet_blocks) {
+        obj.pushKV("signet_challenge",  HexStr(Params().GetConsensus().signet_challenge));
```

Comment:
Sorry, maybe this should use `signetchallenge` to match the argument (here and below)?